manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,17 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+Upon successful completion of the type checking procedure, type information is
+used to create an abstract typed syntax tree (AST). It contains complete
+information about the location and the specific type of each token in the
```

Comment:
`term` rather than `token`: a token is lexical object like `(`, which does not have a type.